keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -408,22 +404,22 @@ apiVersion: resource.k8s.io/v1alpha3
 kind: DeviceClass
 metadata:
   name: acme-gpu
-
-selectors:
-- expression: device.driverName == "gpu.acme.example.com"
-
-config:
-- opaque:
-    driverName: gpu.acme.example.com
-    parameters:
-      apiVersion: gpu.acme.example.com/v1
-      kind: GPUInit
-      # DANGER! This option must not be accepted for
-      # user-supplied parameters.
-      initCommand:
-        - /usr/local/bin/acme-gpu-init
-        - --cluster
-        - my-cluster
+spec:
+  selectors:
+  - cel:
+      expression: device.driverName == "gpu.acme.example.com"
+  config:
+  - opaque:
+      driver: gpu.acme.example.com
+      parameters:
+        apiVersion: gpu.acme.example.com/v1
+        kind: GPUInit
+        # DANGER! This option must not be accepted for
+        # user-supplied parameters.
+        initCommand:
+          - /usr/local/bin/acme-gpu-init
+          - --cluster
+          - my-cluster
 ```
 
 #### Partial GPU allocation
```

Comment:
Could you please expand the description of this section to point out that the partial allocation is not out of the box, the device needs to be exposed as a bunch of sub-devices that can be allocated separately. Otherwise it seems that the scheduler will be able to use the `attributes` or `capacity` on its own to consume a portion of available device property.

Reply:
Thanks for the review! It helps to have someone go through it with fresh eyes.

I've fixed the examples and extended the "Partial GPU allocation". Can you check that it makes more sense now and re-add the LGTM if it does?

Reply:
Absolutely.
/lgtm